app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -196,6 +214,10 @@ private void bindContent() {
         }
     }
 
+    private void removeRefreshing() {
```

Comment:
Refreshing the webpage does not refresh the item, thus it's correct that this does not get executed. You should put this logic in `onProgressChanged` instead.